File: tasks/prac6_python/task1.py
start = int(input("Введите начало диапазона: "))
end = int(input("Введите конец диапазона: "))

total = 0
for i in range(start, end + 1):
    total += i

print(f"Сумма чисел от {start} до {end}: {total}")

File: tasks/prac6_js/task1.js
const prompt = require('prompt-sync')();

const start = parseInt(prompt("Введите начало диапазона: "));
const end = parseInt(prompt("Введите конец диапазона: "));

let total = 0;
for (let i = start; i <= end; i++) {
    total += i;
}

console.log(`Сумма чисел от ${start} до ${end}: ${total}`);